locale/pt-br/texts.po

```diff
@@ -21,25 +79,338 @@ msgstr ""
+
+#: src/screens/ChangePin.js:173
+msgid "Please enter your **new PIN**"
+msgstr "Por favor, entre com o seu **novo PIN**"
+
+#: src/screens/ChangePin.js:186
+msgid "Please enter your **new PIN** again"
+msgstr "Por favor, entre novamente com o seu **novo PIN**"
+
+#: src/screens/ChangePin.js:201
+msgid "New PIN recorded"
+msgstr "Novo PIN salvo"
+
+#: src/screens/ChangePin.js:211
+msgid "CHANGE PIN"
+msgstr "MUDAR PIN"
+
+#: src/screens/ChoosePinScreen.js:124
+msgid "Enter your new PIN code"
+msgstr "Entre com seu novo PIN"
+
+#: src/screens/ChoosePinScreen.js:136
+msgid "Enter your new PIN code again"
+msgstr "Entrei com seu novo PIN novamente"
+
+#: src/screens/ChoosePinScreen.js:68
+msgid "Create a new PIN code,"
+msgstr "Criar um novo PIN,"
+
+#: src/screens/ChoosePinScreen.js:71
+msgid "To confirm the PIN,"
+msgstr "Para confirmar o PIN,"
+
+#: src/screens/ChoosePinScreen.js:150
+msgid "PIN codes don't match. Try again."
+msgstr "Os PINs estão diferentes. Tente novamente."
+
+#: src/screens/ChoosePinScreen.js:173
+msgid "Start the Wallet"
+msgstr "Iniciar a Wallet"
+
+#: src/screens/CreateToken.js:70
+msgid "You don't have any hathor tokens available to create your token."
+msgstr "Você não tem nenhum token Hathor disponível para criar o seu token."
+
+#: src/screens/CreateToken.js:85
+msgid "All fields must be filled."
+msgstr "Todos os campos devem ser preenchidos."
+
+#: src/screens/CreateToken.js:123 src/screens/CreateTokenConfirm.js:84
+msgid "Enter your 6-digit pin to create your token"
+msgstr "Entre com seu PIN de 6 digitos para criar o seu token"
+
+#: src/screens/CreateToken.js:124 src/screens/CreateTokenConfirm.js:85
+msgid "Authorize token creation"
+msgstr "Autorizar criação de token"
+
+#: src/screens/CreateToken.js:140
+msgid "CREATE NEW TOKEN"
+msgstr "CRIAR UM NOVO TOKEN"
+
+#: src/screens/CreateToken.js:146 src/screens/CreateTokenName.js:51
+msgid "Token Name"
+msgstr "Nome do Token"
+
+#: src/screens/CreateToken.js:153
+msgid "Token Symbol"
+msgstr "Símbolo do Token"
+
+#: src/components/TxDetailsModal.js:96 src/screens/CreateToken.js:161
+#: src/screens/PaymentRequestDetail.js:108
+msgid "Amount"
+msgstr "Quantidade"
+
+#: src/screens/CreateToken.js:172
+msgid "Create New Token"
+msgstr "Criar Novo Token"
+
+#: src/screens/CreateTokenAmount.js:109 src/screens/CreateTokenConfirm.js:126
+#: src/screens/CreateTokenDepositNotice.js:39 src/screens/CreateTokenName.js:44
+#: src/screens/CreateTokenSymbol.js:63
+msgid "CREATE TOKEN"
+msgstr "CRIAR TOKEN"
+
+#: src/screens/CreateTokenAmount.js:117
+#, javascript-format
+msgid "Amount of ${ this.name } (${ this.symbol })"
+msgstr "Quantidade de ${ this.name } (${ this.symbol })"
+
+#: src/screens/CreateTokenAmount.js:129
+msgid "Deposit:"
+msgstr "Depósito:"
+
+#: src/screens/CreateTokenAmount.js:133
+#, javascript-format
+msgid "You have ${ amountAvailableText } HTR available"
+msgstr "Você tem ${ amountAvailableText } HTR disponíveis"
```

Comment:
We need plural here

Reply:
I'll skip this one. The problem is that `ngettext` does not support `jt` and `amountAvailableText` is a jsx element.